server/config/config.js

```diff
@@ -0,0 +1,29 @@
+require("env2")("./.env")
+const {
+  NODE_ENV,
+  DEVELOPMENT_DATABASE,
+  PRODUCTION_DATABASE,
+  SECRET,
+  ADMIN_ACCESS_TOKEN,
+} = process.env
+
+const config = env => {
```

Comment:
I like the pattern of `if (...) return ..., if (...) return ..., return ...` to avoid nesting and to keep things less verbose. Kind of like branches of a case statement. Just personal preference though 🙊 

```
const config = env => {
  if (env === production) {
    return {
      database: PRODUCTION_DATABASE,
      secret: SECRET,
    }
  }
  return {
    database: DEVELOPMENT_DATABASE,
    secret: SECRET,
    adminAccessToken: ADMIN_ACCESS_TOKEN,
  }
}
```

Let me know what you think.